cspell.json

```diff
@@ -0,0 +1,27 @@
+{
+  "dictionaries": ["custom-words"],
+  "dictionaryDefinitions": [
+    {
+      "name": "custom-words",
+      "path": "./cspell-wordlist.txt",
+      "addWords": true
+    }
+  ],
+  "ignoreRegExpList": [
+    "/(```+)[\\s\\S]+?\\1/g", // Ignore all words inside of code blocks
```

Comment:
Yeah normally they would cause it to error but cspell still works and even has comments in their examples: https://cspell.org/docs/case-sensitive/#by-glob-pattern

This made me think it was okay, but I can remove it